pulumi/infra/iam/github_oidc.py

```
@@ -0,0 +1,101 @@
+    {{
+      "Effect": "Allow",
+      "Principal": {{
+        "Federated": "{arn}"
+      }},
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {{
+        "StringEquals": {{
+          "token.actions.githubusercontent.com:aud": "sts.amazonaws.com"
+        }},
+        "StringLike": {{
+          "token.actions.githubusercontent.com:sub": "repo:{settings.org}/{repo_name}:ref:refs/heads/{branch}"
```

Ruff (0.14.5)

Function definition does not bind loop variable `branch`

(B023)